### DISCUSSION TITLE
这里为什么没有extras字段，我看数据库里面是有的，可以加上么？

### DISCUSSION THREAD

**Discussion by houming818** [TencentBlueKing/bk-user#778] [Q&A]
https://github.com/TencentBlueKing/bk-user/blob/1ad9537dbefb60e73af0f54796bd2455ae44baad/src/api/bkuser_core/api/login/serializers.py#L26

增加字段extras的支持。

**Comment by wklken** [comment]
extras是有schema的, 是目录默认字段配置声明的name/type等

目前并不对登录upsert这种场景开放(因为没有对应的校验逻辑

并且这块打算重构, 下掉`/api/v1/login`接口)
